app/Services/LearningActivityProducingExportBuilder.php

```diff
@@ -42,7 +42,17 @@ public function getJson(array $learningActivities, ?int $limit = null): string
                 'status'          => $this->translator->get('general.'.strtolower($activity->status->status_label)),
                 'url'             => route('process-producing-edit', ['id' => $activity->lap_id]),
                 'chain'           => $this->formatChain($activity),
-                'feedback'        => $activity->feedback->fb_id ?? null,
+                'feedback'        => [
```

Comment:
I don't think the null coalescing here will ever happen as an array will always be created. Perhaps extract this part into a seperate method and check whether or not a feedback is attached to the activity. If not, return null. If there is a feedback attached, create the array and return it.